brunofurmon/mayt-api
# Initial webserver

# What this PR does?

- Adds initial web server structure

## - Before/After

- There was nothing. Now there is something.

## - How to test it?

- `yarn setup && yarn run-dev` should start a web server listening on port 3000 (default, configured on `.env` file)

## - TODOs

- None

## - Risks

- None
